### PR TITLE
[v6r18] Mostly pylint fixes

### PR DESCRIPTION
**Pull request by fstagni**

### CODE REVIEW

File: DataManagementSystem/Client/FTSJob.py [hunk 1]
```diff
@@ -35,14 +35,14 @@
 from DIRAC.Resources.Catalog.FileCatalog     import FileCatalog
 from DIRAC.Core.Utilities.ReturnValues import returnSingleResult
 
-import fts3.rest.client.easy as fts3
+import fts3.rest.client.easy as fts3 #pylint: disable=import-error
```
Comment (chaen) [inline DataManagementSystem/Client/FTSJob.py]: why do you want to ignore that ?

File: DataManagementSystem/Client/FTSJob.py [hunk 2]
```diff
@@ -35,14 +35,14 @@
...
 
 # We can't use the default pycurl because of known bugs
 # memory leak: https://bugzilla.mozilla.org/show_bug.cgi?id=1202413
 # SIGALRM handling: https://curl.haxx.se/mail/lib-2008-09/0197.html
 # So we need to use the Request module. The correct version is available
 # from fts-rest v3.5.2 or in the lcg-bundle 2017-01-27
-from fts3.rest.client.request import Request as ftsSSLRequest
+from fts3.rest.client.request import Request as ftsSSLRequest #pylint: disable=import-error
```
Comment (chaen) [inline DataManagementSystem/Client/FTSJob.py]: I *think* that it is not necessary anymore. I thought I had removed it, I have to check

File: DataManagementSystem/Client/FTSJob.py (hunk 1 above)
Comment (fstagni) [inline DataManagementSystem/Client/FTSJob.py]: Because there's no `pip install fts3` so this is shown as an error by pylint tests :-/

Comment (chaen) [inline DataManagementSystem/Client/FTSJob.py]: of course there is :-)
pip install "git+https://gitlab.cern.ch/fts/fts-rest.git@master"
Just replace master with the version you want

**Pull request merged**